packages/search/addon/components/navi-search-result/report.hbs

```diff
@@ -0,0 +1,12 @@
+<div {{did-insert this.search @query}}>
```

Comment:
©️

Reply:
Can we remove this div and move the `did-insert` into an init method in the component?